workflows/job_sender.json

```diff
@@ -15,7 +15,7 @@
       "parameters": {
         "queue": "job_queue",
         "sendInputData": false,
-        "message": "={\n    \"job\": [\n        {\n            \"type\": \"download\",\n            \"datasetURI\": \"https://somethin\"\n        },\n        {\n            \"type\": \"extract\"\n        }\n    ],\n    \"emailAddress\": \"peter@tosh.com\"\n}",
+        "message": "={\n    \"job\": [\n        {\n            \"id\": 123,\n            \"type\": \"download\",\n            \"inputs\": [\n              \"https://opendata.dwd.de/climate_environment/CDC/derived_germany/soil/daily/historical/derived_germany_soil_daily_historical_1001.txt.gz\"\n            ]\n        },\n        {\n            \"id\": 456,\n            \"type\": \"extract\",\n            \"inputs\": [\n                {\n                    \"outputOfId\": 123,\n                    \"outputIndex\": 0\n                }\n            ]\n        }\n    ],\n    \"emailAddress\": \"peter@tosh.com\"\n}",
```

Comment:
Is the specified DWD file always accessible?

Reply:
I hope so. Time will tell